tests/forcefields/test_utils.py

```diff
@@ -163,3 +163,27 @@ def test_fix_symmetry(fix_symmetry):
         assert symmetry_init["number"] == symmetry_final["number"] == 229
     else:
         assert symmetry_init["number"] != symmetry_final["number"] == 99
+
+
+def test_m3gnet_pot():
+    import matgl
+    from matgl.ext.ase import PESCalculator
+
+    m3gnet_calculator = ase_calculator(
```

Comment:
I actually want to implement it as 
```python
    m3gnet_calculator = ase_calculator(
        calculator_meta="MLFF.M3GNet",
        **{"path": "M3GNet-MP-2021.2.8-DIRECT-PES", "stress_weight": 2.0},
    )

    m3gnet_default = ase_calculator(  # uses "M3GNet-MP-2021.2.8-PES" per default
        calculator_meta="MLFF.M3GNet", **{"stress_weight": 2.0},
    )
```

and the test passes, but ruff-format changes it....

Reply:
Just define the two dicts

Reply:
yup. Just noticed it after pushing 🤣 so I didn't want you to merge it like that